Source/WTF/wtf/RandomNumber.h

```diff
@@ -25,15 +25,25 @@
 
 #pragma once
 
+#include <cstdint>
+
 namespace WTF {
 
 // Returns a cryptographically secure pseudo-random number in the range [0, 1).
 WTF_EXPORT_PRIVATE double randomNumber();
 
+// Returns a cryptographically secure pseudo-random number in the range (0, UINT_MAX].
+WTF_EXPORT_PRIVATE unsigned cryptographicallyRandomUint32();
+
+// Returns a cryptographically secure pseudo-random number in the range (0, UINT64_MAX].
+WTF_EXPORT_PRIVATE uint64_t cryptographicallyRandomUint64();
```

Comment:
Really glad you did this! I have been sick so missed my chance to review. I believe you already landed this, but:

- I would capitalize the "I" for "integer" and called them `cryptographicallyRandomUInt32` and `64`
- I would use the type `uint32_t` instead of `unsigned`, even though this is really unimportant and unclear when we should do that
- I would have stuck with the naming in this header and just called these `randomUInt32` and `randomUInt64`
- Or I would have renamed `randomNumber` to `cryptographicallyRandomDouble`, since it’s not so widely used we can’t rename
- I would also want to audit callers of `cryptographicallyRandomDouble` to see if they are really OK, since I am not sure it has enough random bits in it, not really a "random double" in my view